### PR TITLE
Extended Tweet Issues 56

### PR DESCRIPTION
**Pull request by steve-winter**
Enable search to return full_text of tweet.

### CODE REVIEW

File: twitter/search.go [hunk 1]
```diff
@@ -49,6 +49,7 @@ type SearchTweetParams struct {
 	MaxID           int64  `url:"max_id,omitempty"`
 	Until           string `url:"until,omitempty"`
 	IncludeEntities *bool  `url:"include_entities,omitempty"`
+	TweetMode       string `url:"tweet_mode,omitempty"`
```
Comment (shurcooL) [inline twitter/search.go]: Where can a user of this Go package find out what string values `TweetMode` field can be set to?

Is it available in Twitter API docs?

Comment (shurcooL) [inline twitter/search.go]: I'm not seeing it mentioned at https://dev.twitter.com/rest/reference/get/search/tweets, am I looking in the wrong place?

Comment (steve-winter) [inline twitter/search.go]: The documentation only seems to exist here; https://dev.twitter.com/overview/api/upcoming-changes-to-tweets.

File: twitter/statuses.go [hunk 2]
```diff
@@ -12,6 +12,7 @@ import (
 // Deprecated fields: Contributors, Geo, Annotations
 type Tweet struct {
 	Coordinates          *Coordinates           `json:"coordinates"`
+	FullText             string                 `json:"full_text"`
```
Comment (dghubble) [inline twitter/statuses.go]: Isn't full_text intended to be a subfield of extended_tweet? What tweet did you test this on?

Comment (dghubble) [inline twitter/statuses.go]: I see, this is assuming extended mode. We'll want to support both modes.

**Comment by dghubble** [comment]
Included in #63, thanks!

**Pull request closed without merging**